parsers/deathloopvaccine.py

```diff
@@ -0,0 +1,286 @@
+import datetime
+import re
+import os
+import signal
+
+import psutil
+
+from datetime import datetime
+from helpers import Parser, config
+
+
+#
+# simple utility to prevent Everquest Death Loop
+#
+# The utility functions by parsing the current (most recent) Everquest log file, and if it detects
+# Death Loop symptoms, it will respond by initiating a system process kill of all "eqgame.exe"
+# processes (there should usually only be one).
+#
+# We will define a death loop as any time a player experiences X deaths in Y seconds, and no player
+# activity during that time.  The values for X and Y are configurable, via the DeathLoopVaccine.ini file.
+#
+# For testing purposes, there is a back door feature, controlled by sending a tell to the following
+# non-existent player:
+#
+#   death_loop:     Simulates a player death.
+#
+#                   Note however that this also sets a flag that disarms the conceptual
+#                   "process-killer gun", which will allow every bit of the code to
+#                   execute and be tested, but will stop short of actually killing any
+#                   process
+#
+#                   The "process-killer gun" will then be armed again after the simulated
+#                   player deaths trigger the simulated process kill, or after any simulated
+#                   player death events "scroll off" the death loop monitoring window.
+#
+class DeathLoopVaccine(Parser):
```

Comment:
The name is cute but we may want to rename this something clearer like "DeathLoopPrevention", IDK.
Vaccines are apparently political now in 2022 or something XD

Reply:
Up to you, I'm ambivalent.

Reply:
What's your thoughts here?  If you wish to change it, I'll bang it out with some other changes I'm in the middle of doing.